server.js

```diff
@@ -1,90 +1,32 @@
 'use strict';
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const apiRouter = require('./routes/api.routes');
+const port = process.env.PORT || 3000;
 
-var express = require('express');
-var mongo = require('mongodb');
-var mongoose = require('mongoose');
-var bodyParser = require('body-parser');
-const dns = require('dns');
+// Configurations
+require('./config/db.config');
 
-var cors = require('cors');
-
-var app = express();
-
-// Basic Configuration 
-var port = process.env.PORT || 3000;
-
-/** this project needs a db !! **/ 
-mongoose.connect(process.env.MONGOLAB_URI || "mongodb://scorpion9979:ahmedFcc221@ds161092.mlab.com:61092/urlshortener");
+const app = express();
 
 app.use(cors());
-
-/** this project needs to parse POST bodies **/
 app.use(bodyParser.urlencoded({extended: false}));
-
 app.use('/public', express.static(process.cwd() + '/public'));
 
-app.get('/', function(req, res){
-  res.sendFile(process.cwd() + '/views/index.html');
-});
+// Routes
+app.use('/api', apiRouter);
 
-  
-// your first API endpoint... 
-app.get("/api/hello", function (req, res) {
-  res.json({greeting: 'hello API'});
-});
-
-var Schema = mongoose.Schema;
-var urlSchema = new Schema({
-  original_url: {type: String, required: true},
-  short_url: {type: String, required: true}
-});
-var Model = mongoose.model("Model", urlSchema);
+//@INFO: could create a new home.routes.js and home.controller.js if we had more functions
+app.get('/', function(req, res){ res.sendFile(process.cwd() + '/views/index.html') });
```

Comment:
This could have been something like
`app.use('/', homeRouter)` that would have its own `homeController.showHome()` logic